deployment/production/docker/headless/REQUIREMENTS.txt

```diff
@@ -0,0 +1,8 @@
+requests>=2.6.2
+tzlocal==1.2
+hammock==0.2.4
+nose2==0.6.5
+coverage==4.4.2
+pyinotify==0.9.6
+celery==4.1.0
+mock==2.0.0
```

Comment:
do we need mock, coverage, nose2, hammock in production?

Reply:
Maybe we won't need all of this, you are right.